djangoevents/tests/test_domain.py

```diff
@@ -95,3 +99,18 @@ def test_events_dont_have_schema_version_when_disabled():
 def test_events_have_schema_version_when_enabled():
     event = SampleEntity.Created(entity_id=1)
     assert event.schema_version == 1
+
+
+@override_settings(DJANGOEVENTS_CONFIG={
+    'ADDS_SCHEMA_VERSION_TO_EVENT_DATA': True,
+})
+def test_events_have_correct_schema_version():
+    original_version = getattr(SampleEntity.Created, 'version', None)
+    expected_version = 666
+
+    try:
+        SampleEntity.Created.version = expected_version
+        event = SampleEntity.Created(entity_id=1)
+        assert event.schema_version == expected_version
+    finally:
+        SampleEntity.Created.version = original_version
```

Comment:
I'd prefer using `mock.patch.object` instead. Currently it's hard to tell what test does at first glance. I'll prepare a patch.